ycmd/completers/go/tests/gocode_completer_test.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
Good idea, will do. Do you have a handy example in one of the other tests for doing the E2E?